service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -43,7 +43,8 @@ class ScaladslKafkaApiSpec extends WordSpecLike with Matchers with BeforeAndAfte
         "akka.remote.netty.tcp.port" -> "0",
         "akka.remote.netty.tcp.hostname" -> "127.0.0.1",
         "akka.persistence.journal.plugin" -> "akka.persistence.journal.inmem",
-        "akka.persistence.snapshot-store.plugin" -> "akka.persistence.snapshot-store.local"
+        "akka.persistence.snapshot-store.plugin" -> "akka.persistence.snapshot-store.local",
+        "lagom.services.kafka_native" -> s"tcp://localhost:${KafkaLocalServer.DefaultPort}"
```

Comment:
I don't think it really matters - this was just putting Kafka in the integration tests.